packages/core/data-transfer/lib/providers/local-file-destination-provider/index.ts

```diff
@@ -255,6 +255,7 @@ class LocalFileDestinationProvider implements IDestinationProvider {
 
         entry
           .on('finish', () => {
+            console.log('FINISH WRITING ALREADY');
```

Comment:
:eyes:

Reply:
If it didn't want to get yelled at, it should have finished writing faster. It knew what it was doing.